contracts/GNTAllocation.sol

```diff
@@ -53,19 +58,17 @@ contract GNTAllocation {
         if (tokensCreated == 0) {
             // Fetch number of total tokens locked.
             tokensCreated = gnt.balanceOf(this);
+            tokensPerShare = tokensCreated / SHARES;
```

Comment:
It will be better to calculate "leftovers" straight away and replace `tokensCreated` entry. Also update comment.